clouddriver-ecs/src/main/java/com/netflix/spinnaker/clouddriver/ecs/controllers/EcsClusterController.java

```diff
@@ -37,4 +39,10 @@ public EcsClusterController(EcsClusterProvider ecsClusterProvider) {
   public Collection<EcsCluster> getAllEcsClusters() {
     return ecsClusterProvider.getAllEcsClusters();
   }
+
+  @RequestMapping(value = {"/ecs/getAllEcsClusterDetails/{account}/{region}"})
```

Comment:
instead of getAll you just want to call it ecsClusterDetails (see ecsClusters and getAllEcsClusters for reference.

`"/ecs/ecsClusterDetails/{account}/{region}"`